external-crates/move/crates/move-compiler/src/sui_mode/typing.rs

```diff
@@ -916,6 +913,18 @@ fn exp(context: &mut Context, e: &T::Exp) {
     }
 }
 
+fn otw_special_cases(context: &Context) -> bool {
+    for token in ["btc", "eth", "usdc", "usdt"] {
+        if context.current_module().value.is(BRIDGE_ADDR_NAME, token) {
```

Comment:
nit, just a bit more idiomatic 
`["btc", "eth", "usdc", "usdt"].iter().any(|token| context.current_module().value.is(BRIDGE_ADDR_NAME, token))`